debian/postinst

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+ldconfig
```

Comment:
Should probably support the command arguments.  Only run `ldconfig` if `$1` is `configure`.  Example:

https://github.com/DaGoodBoy/webpy-example/blob/master/debian/postinst

Reply:
👍